src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -1115,16 +1094,23 @@ k
             }
```

Comment:
if anything these can come from the `InputController`. but, yeah, **_all_** the `prop`s should have been removed some time ago.